lib/livebook_web/live/session_live/live_widget/vega_lite_live.ex

```diff
@@ -0,0 +1,32 @@
+defmodule LivebookWeb.SessionLive.LiveWidget.VegaLiteLive do
```

Comment:
I think perhaps it is fine to namespace this as:

```suggestion
defmodule LivebookWeb.LiveWidget.VegaLiteLive do
```

Sure, we only use it in `SessionLive` but there is nothing in this specific to SessionLive? 🤔

Reply:
Good point!